tests/python_compiler/transforms/mbqc/test_xdsl_convert_to_mbqc_formalism.py

```diff
@@ -272,10 +298,10 @@ def test_rotxzx_gate(self, run_filecheck):
     def test_cnot_gate(self, run_filecheck):
         """Test for lowering a CNOT gate to a MBQC formalism."""
         program = """
-            func.func @test_func() {
-                // CHECK: [[q1:%.+]] = "test.op"() : () -> !quantum.bit
-                // CHECK: [[q9:%.+]] = "test.op"() : () -> !quantum.bit
+            func.func @test_func(%arg0 : f64) {
```

Comment:
minor but I guess we don't need `%arg0` here if we aren't adding the global phase